azurerm/resource_arm_dev_test_lab.go

```diff
@@ -0,0 +1,210 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/Azure/azure-sdk-for-go/services/devtestlabs/mgmt/2016-05-15/dtl"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmDevTestLab() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDevTestLabCreateUpdate,
+		Read:   resourceArmDevTestLabRead,
+		Update: resourceArmDevTestLabCreateUpdate,
+		Delete: resourceArmDevTestLabDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateDevTestLabName(),
```

Comment:
Would this be better suited to be in the `azure` package with the other validators?